.env.default

```diff
@@ -14,6 +14,9 @@ GOOGLE_CLIENT_SECRET=
 # Will be set automatically by the server
 FLASK_SECRET_KEY=
 
+# Comma separated list of domains.
+CORS_HOSTS=http://localhost:8080
```

Comment:
Another small thing, but I think changing this to something like `ALLOWED_ORIGINS` or something would be better. Basically something that describes that only these domains can make requests to the server. Maybe update the comment to reflect that too.

Reply:
I like `ALLOWED_ORIGINS`. Is there a technical name for this that we ought to use instead of either of these options?

Reply:
Googling around a bit, it seems that "allowed origins" is the right term for this. Will change.

Reply:
See ce023f47e03dba6e048bc62c37dfcaa1781cf1ca.